### DISCUSSION TITLE
V1.3.2 UBL - BLtouch

### DISCUSSION THREAD

**Discussion by ACGLaguiole** [Jyers/Marlin#709] [Ideas]
Hello!

Firstly I just want to say thanks for this firmware, it's SO GOOD! The M48 addition in v1.3.2 was the best addition due to it gives the possibilty to "live" check and fault search the BLtouch and it's potential problems.

I had alot of issues with my BLtouch, would it be interesting if I wrote a sort of a white paper to put on the wiki? I would love to contribute to this community. Also if there is any other interested lets cooperate. I would especially need help from people who is profficient in the source code (what happens behind the hood) and mechanical engineering (squaring/aligning and potential tolerances within the system). I am myself an electronics engineer.

Cheers from Sweden!

**Comment by Jyers** [comment]
That would be a great addition! Feel free to either post any progress here or send me a draft at Jyers.Marlin@gmail.com

**Reply by Johnny-Bones** [reply]
Since we're on the topic of wiki's, is there any place i can find a manual of sorts that explains what each new/old setting in 1.3.2 does other than the standard wiki? I'm not really sure what the "Leveling Active" setting does. I mean, i know what it implies it does, but a detailed explanation of some of these settings would he extremely useful for people like me that are new to abl's in general. Thanks for making this awesome firmware man!

**Comment by darco415** [comment]
How do you access the M48 thru the menu?

**Reply by Jyers** [reply]
Its in advanced>probe

**Reply by darco415** [reply]
I compile my own fw using your latest templates...for some reason the m48 did not appear after latest compile. I tried your pre config fw and of course its there...is there a specific line I can check? thxs

**Reply by Jyers** [reply]
You need to have `Z_MIN_PROBE_REPEATABILITY_TEST` enabled in your config

**Reply by darco415** [reply]
its always been enabled...

**Reply by adam3654** [reply]
> its always been enabled...

![iRtrBOr.png](https://user-images.githubusercontent.com/16920869/117335740-786adb00-ae69-11eb-9684-806d14368966.png)

**Reply by Jyers** [reply]
> its always been enabled...

Maybe clear eeprom?

**Reply by darco415** [reply]
ok all sorted..... I have two local repos...I thought both were updated to latest....compiled from latest repo...all good, thxs bud
